packages/sw-runtime-caching/test/sw/request-wrapper.js

```diff
@@ -0,0 +1,186 @@
+importScripts('setup.js');
+
+describe('Test of the RequestWrapper class', function() {
+  const CACHE_NAME = location.href;
+  const CACHE_WILL_UPDATE_PLUGIN = {cacheWillUpdate: () => {}};
+  const CACHE_WILL_MATCH_PLUGIN = {cacheWillMatch: () => {}};
+  const CACHED_URL = '/cached';
+
+  let globalStubs = [];
+
+  before(() => {
+    return caches.delete(CACHE_NAME)
+      .then(() => caches.open(CACHE_NAME))
+      .then((cache) => cache.put(CACHED_URL, new Response('')));
```

Comment:
Again this feels odd doing only at the beginning of the test suite. would make sense to move to beforeEach (Just to make sure one test doesn't mess up the rest).